### ISSUE TITLE
[JAWS, IE11] Holding down arm/legs arrow keys can duplicate announcements

### ISSUE THREAD

**Issue by phet-steele** [phetsims/john-travoltage#227]
Holding down arrow keys to move the leg/arm queues up multiple messages that all repeat the same position value. For example,

1. Start the leg at position 30. Hold down the left key until position 0.
2. You will hear ~29 messages (one for each position between 0 and 30) BUT each message will say "Position 0.....".
3. I never hear any electron updates while doing this with the leg (not sure about discharge messages).

While holding down an arrow key, JAWS is silent. I think this is fine, but different than VO, which gives you a position update when the previous position announcement finishes. Only once I release an arrow key do I get any messages in JAWS.

For phetsims/tasks/issues/813.
URL: http://www.colorado.edu/physics/phet/dev/html/john-travoltage/1.3.0-dev.20/john-travoltage_en.html
Version: 1.3.0-dev.20 2017-04-05 17:45:45 UTC
Features missing: touch, fullscreen
Flags: pointerEnabled, msPointerEnabled, pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; WOW64; Trident/7.0; Touch; .NET4.0C; .NET4.0E; .NET CLR 2.0.50727; .NET CLR 3.0.30729; .NET CLR 3.5.30729; Tablet PC 2.0; rv:11.0) like Gecko
Language: en-US
Window: 1536x774
Pixel Ratio: 2.5/1
WebGL: WebGL 0.94
GLSL: WebGL GLSL ES 0.94
Vendor: Microsoft (Internet Explorer)
Vertex: attribs: 16 varying: 15 uniform: 512
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 16384x16384
OES_texture_float: true
Dependencies JSON: {"assert":{"sha":"a707328c","branch":"master"},"axon":{"sha":"f7beced2","branch":"master"},"babel":{"sha":"349f2b73","branch":"master"},"brand":{"sha":"054ca5c3","branch":"master"},"chipper":{"sha":"7c6e001f","branch":"master"},"dot":{"sha":"569939e1","branch":"master"},"john-travoltage":{"sha":"d3c5efa2","branch":"master"},"joist":{"sha":"d11d93da","branch":"master"},"kite":{"sha":"81166ce9","branch":"master"},"phet-core":{"sha":"c5c6c2a8","branch":"master"},"phetcommon":{"sha":"85801e7b","branch":"master"},"query-string-machine":{"sha":"d8a4ff18","branch":"master"},"scenery":{"sha":"e5fadaaf","branch":"master"},"scenery-phet":{"sha":"2cc12884","branch":"master"},"sherpa":{"sha":"ad343771","branch":"master"},"sun":{"sha":"cf75fc6d","branch":"master"},"tandem":{"sha":"cc7876d0","branch":"master"},"vibe":{"sha":"549baf54","branch":"master"}}

**Comment by jessegreenberg** [comment]
Thanks @phet-steele, there is a chance that this was fixed by #210, I will check.

**Comment by jessegreenberg** [comment]
From #210 electron updates come through once it finishes, but point 2 above is still happening. Time for a JSFiddle.

**Comment by jessegreenberg** [comment]
As far as I can tell, this is a JAWS bug for all sliders.  JAWS is doing the same thing with this example:
https://jsfiddle.net/1j1y3s8p/

And if you remove the aria value text attribute (https://jsfiddle.net/1j1y3s8p/1/) JAWS is doing this when reading out the value. I will do some google searching.

**Comment by jessegreenberg** [comment]
@emily-phet do you think we should spend more time on this bug? Perhaps submit a report to Freedom Scientific and/or run it by friends at the IDRC?

**Comment by emily-phet** [comment]
@jessegreenberg Ashley is a good go-to person for JAWS. Let's ask her to take a look at a version of John Travoltage, and move the objects by stepping through individually and by holding down the arrow keys and see if she thinks the behavior seems unexpected. I think we would all agree the behavior is less than ideal, but if it's a JAWS bug the behavior may not be surprising. My guess is that it won't be too surprising, and if that's the case we can just submit a bug report and not try to find a workaround ourselves.

**Comment by jessegreenberg** [comment]
@emily-phet that sounds great, marking as ready for review. If Ashley doesn't find an issue with this we will mark as "wontfix" and follow up with an issue to VFO.

**Comment by jessegreenberg** [comment]
Marking as wontfix since we are letting this go as a screen reader bug.

**Comment by jessegreenberg** [comment]
This  has been added to https://docs.google.com/document/d/1518zv6F0odExFsodShZxwTmWNQk3civuNwtQUR-rEBs/edit so there is no need to keep this issue open any more.

**Comment by lmulhall-phet** [comment]
Just to further document this issue:

1. This is JAWS-specific. Has nothing to do with browser.
2. I made a video:

https://drive.google.com/open?id=1Cus9PGAZweIeE6BWTNI4fwNaASFVzLuv
